static/app/components/multiPlatformPicker.tsx

```diff
@@ -144,7 +161,7 @@ function PlatformPicker(props: PlatformPickerProps) {
       });
     }
   }, DEFAULT_DEBOUNCE_DURATION);
-
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@vpandiarajan20 So while you didn't modify this code, I think it would be nice to actually fix the warning instead of adding the eslint-disable comment.